yellowbrick/target/class_balance.py

```diff
@@ -325,7 +337,11 @@ def class_balance(
 
     # Fit and transform the visualizer (calls draw)
     visualizer.fit(y_train, y_test)
-    visualizer.finalize()
```

Comment:
What happens in the case when `y_test`is `None`? Does `ClassBalance` gracefully ignore it?

Reply:
Yes; I took another look at the tests and the one that we have for the quick method tests this case explicitly:
```python
    def test_quick_method(self):
        """
        Test the quick method producing a valid visualization
        """
        dataset = make_fixture(binary=False, split=False)

        viz = class_balance(dataset.y)

        assert isinstance(viz, ClassBalance)
        self.assert_images_similar(viz, tol=0.5)
```
So I added another quick method test for the other case, where there *are* train and test splits, just to be safe! Here's how it looks with splits:

![test_quick_method_with_splits](https://user-images.githubusercontent.com/8760385/74089832-579fb400-4a73-11ea-91cf-fccc0beaa9ff.png)

and without:

![test_quick_method](https://user-images.githubusercontent.com/8760385/74089836-60908580-4a73-11ea-8323-b853eed5a621.png)